jenkins-pipelines/operator/eks/longevity-scylla-operator-3h-eks-backup.jenkinsfile

```diff
@@ -6,7 +6,7 @@ longevityPipeline(
     backend: 'k8s-eks',
     region: 'eu-north-1',
     test_name: 'longevity_test.LongevityTest.test_custom_time',
-    test_config: 'test-cases/scylla-operator/longevity-scylla-operator-3h-backup.yaml',
+    test_config: '''["test-cases/scylla-operator/longevity-scylla-operator-3h-backup.yaml", "configurations/operator/expose-pod-ips.yaml"]''',
```

Comment:
Why is it needed?

Bug reference?

Need to add details to the commit message.